.github/workflows/ci.yml

```diff
@@ -23,16 +23,12 @@ jobs:
         with:
           components: rustfmt, clippy
       - uses: Swatinem/rust-cache@v2
-      - name: Install Pixi
-        run: |
-          curl -fsSL https://pixi.sh/install.sh | bash
-          echo "$HOME/.pixi/bin" >> $GITHUB_PATH
-          echo "GDAL_HOME=$(pwd)/build/.pixi/envs/default" >> "$GITHUB_ENV"
-          echo "LD_LIBRARY_PATH=$(pwd)/build/.pixi/envs/default/lib" >> "$GITHUB_ENV"
-          echo "GEOS_LIB_DIR=$(pwd)/build/.pixi/envs/default/lib" >> "$GITHUB_ENV"
-          # TODO: infer from toml file/lockfile
-          echo "GEOS_VERSION=3.12.1" >> "$GITHUB_ENV"
-          echo "PKG_CONFIG_PATH=$(pwd)/build/.pixi/envs/default/lib/pkgconfig" >> "$GITHUB_ENV"
```

Comment:
I'm not sure but we may still need these environment variables so that the Rust build tooling knows where to find libraries. But maybe `activate-environment: true` will be sufficient

Reply:
yuuup ... we're in good ol' debug-by-push at the moment 🙄 :-)

I'm going to bet most of the stuff is set correctly by the environment, but we might need to suppliment.